lit_gpt/config.py

```diff
@@ -602,6 +601,198 @@ def norm_class(self) -> Type:
 ]
 configs.extend(freewilly_2)
 
+code_llama = [
```

Comment:
I haven't looked into this deeply, but don't we need to make the rope theta configurable? https://huggingface.co/codellama/CodeLlama-34b-hf/blob/main/config.json#L19 sets it to `1000000` but we use `10000`: https://github.com/Lightning-AI/lit-gpt/blob/main/lit_gpt/model.py#L299

cc @rasbt